plugin/hover.py

```diff
@@ -82,17 +86,28 @@ def run(
         point: Optional[int] = None,
         event: Optional[dict] = None
     ) -> None:
-        temp_point = point
-        if temp_point is None:
+        range_hover_provider = False
+        session = self.best_session(self.range_hover_capability)
+        if session:
+            capability = session.get_capability(self.range_hover_capability)
+            if isinstance(capability, bool):
+                range_hover_provider = capability
```

Comment:
It's not correct to just get the "best_session" since it won't be accurate with multiple session running. The check should be later, when iterating sessions.